modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionDemandMessage.java

```diff
@@ -179,7 +179,9 @@ Object topic() {
 
     /**
      * @param topic Topic.
+     * @deprecated Useless.
```

Comment:
I think we should describe here, why it is not used anymore.
Since it's only internal API, should we just remove it without deprecation?

Reply:
Removal will cause incompatibility with older nodes.

Reply:
I think we should place exactly your words for deprecated annotation description.

Reply:
Let's leave this to future refactoring.